src/backend/Dockerfile

```diff
@@ -24,10 +39,22 @@ WORKDIR /usr/src/app
 
 COPY pyproject.toml poetry.lock environment.yaml ./
 COPY third-party ./third-party
-RUN /opt/poetry/bin/poetry config virtualenvs.create false && \
-    /opt/poetry/bin/poetry install
+RUN /opt/poetry/bin/poetry config virtualenvs.create false
+# Copy poetry deps from the build image
+COPY --from=build /usr/local/lib/python3.9/site-packages /usr/local/lib/python3.9/site-packages
```

Comment:
This is just an attempt to trim down our image a little bit, borrowing from some of the tricks used in the nexstrain/base image build.